tests/components/rtsptowebrtc/test_config_flow.py

```diff
@@ -0,0 +1,118 @@
+"""Test the RTSPtoWebRTC config flow."""
+
+from __future__ import annotations
+
+from typing import Any
+from unittest.mock import patch
+
+import rtsp_to_webrtc
+
+from homeassistant import config_entries, setup
+from homeassistant.components.rtsptowebrtc import DOMAIN
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+CONFIG: dict[str, Any] = {}
+
+
+async def test_web_full_flow(hass: HomeAssistant):
+    """Check full flow."""
+    assert await setup.async_setup_component(hass, DOMAIN, CONFIG)
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result.get("type") == "form"
+    assert result.get("step_id") == "user"
+    assert result.get("data_schema").schema.get("server_url") == str
+    assert not result.get("errors")
+    assert "flow_id" in result
+    with patch("rtsp_to_webrtc.client.Client.heartbeat"):
+        result = await hass.config_entries.flow.async_configure(
```

Comment:
We normally patch the integration setup function on all create entry results to avoid setting up the whole integration.

Reply:
I've added a patch to async_setup_entry, then asserted that is was invoked.

(Note: This existing heartbeat patch is This is actually for work happening in the configuration flow to verify the server is healthy, which mirrors the integration setup)